docs/scos/dev/front-end-development/202304.0/migration-guide-upgrade-nodejs-to-v18.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Migration guide - Upgrade Node.js to v18 and npm to v9
+last_updated: May 16, 2023
+description: Use this guide to upgrade Node.js to v18 and npm to v9.
+template: concept-topic-template
+---
+
+This document provides instructions for upgrading Node.js to v18 and npm to v9.
+
+## Overview
+
+According to [Node.js schedule](https://github.com/nodejs/release#release-schedule) the 16 version will be maintained till 11.09.2023. So Spryker recommends migrating to 18 LTS version.
+
+*Estimated migration time: 1h*
+
+## 1) Update configuration files
+
+1. Set up new versions of `node/npm` in the main `*.yml` files, like `deploy.yml`, `deploy.dev.yml`, and `deploy.ci.yml`:
+
+```yaml
+image:
+    ...
+    node:
+        version: 18
+        npm: 9
+```
+
+{% info_block infoBox "Note" %}
+
+To ensure the CI jobs run successfully, add the same config part to all `deploy.*.yml` files used by the frontend.
+
+{% endinfo_block %}
+
+2. In the root directory, create/update `.nvmrc` file with the following content:
+
+```text
+18.16.0
+```
+
+3. Update or add dependencies and engines in the `package.json`:
+
+```json
+{
+    ...
+    "engines": {
+        "node": ">=18.0.0",
+        "npm": ">=9.0.0"
+    },
+    ...
+    "devDependencies": {
+        ...,
+        "@babel/preset-env": "~7.20.2",
+        "@spryker/oryx-for-zed": "~3.0.0",
+        "autoprefixer": "~10.4.14",
+        "postcss": "~8.4.24",
+        "postcss-loader": "~7.3.0",
+        "sass-resources-loader": "~2.2.5",
+        ...
+    },
+    ...
+}
+```
+
+4. Remove dependency in the `package.json`:
+
+```json
+    "@spryker/sass-resources-loader": "x.x.x"
+```
+
+5. Update webpack config in `frontend/configs/development.js`:
+  - Replace `options` of `postcss-loader`;
```

Comment:
If these are steps, then use the numbered list instead of bulleted points. Also, correct grammar:
```suggestion
  1. In `postcss-loader`, replace `options`.
```